lib/horizon/order_book/price.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Stellar.Horizon.OrderBook.Price do
+  @moduledoc """
+  Represents a `Price` for an order book.
+  """
+
+  @behaviour Stellar.Horizon.Resource
+
+  alias Stellar.Horizon.Mapping
+
+  @type t :: %__MODULE__{
+          price_r: map(),
+          price: float(),
+          amount: float()
+        }
+
+  defstruct [
+    :price_r,
+    :price,
+    :amount
+  ]
+
+  @price_r_mapping [
+    n: :integer,
```

Comment:
@MrJuanGaviriaK what is a `price_r_mapping`? IMO, you can remove this constant and place it directly in `@mapping`